src/sheets/quadrone/actor/parts/ActorPortrait.svelte

```diff
@@ -71,13 +71,29 @@
   class={['actor-image', currentPortraitShape, { dead: actorIsDead }]}
   style="position: relative;"
 >
-  <img
-    src={imageUrl}
-    alt={imageAlt}
-    class={['pointer', { dead: actorIsDead }]}
-    data-action={context.unlocked ? 'editImage' : 'showArtwork'}
-    data-edit={context.portrait.path}
-  />
+  {#if context.portrait.isVideo}
+    <video
+      src={imageUrl}
+      autoplay
+      bind:paused={actorIsDead}
+      loop
+      muted
+      playsinline
+      disablepictureinpicture
+      class={['pointer', { dead: actorIsDead }]}
+      data-action={context.unlocked ? 'editImageVideo' : 'showArtwork'}
+      data-edit={context.portrait.path}
+      title={imageAlt}
+      >{context.portrait.truePath ?? imageUrl}</video>
```

Comment:
I wanted this to be fallback, but because of the data-edit, the innerHTML has to be the path

Reply:
I think we're good with this. Doesn't seem to be hurting anything.